content/api/plugins/before-run-api.md

```diff
@@ -74,7 +74,7 @@ module.exports = (on, config) => {
     //     // ... more specs
     //   ],
     //   specPattern: [
-    //     '**/*_spec.js'
+    //     '**/*.cy.{js,jsx,ts,tsx}'
```

Comment:
The `specPattern` won't default to that value if the `specs` array contains a spec like `relative: 'cypress/integration/login_spec.js'`. This example might need to be double-checked.

Reply:
Agreed. I wasn't certain on this one and forgot to add that note to the PR. I'll check it out when we get the dev build.